src/hierarchy.ts

```diff
@@ -275,10 +276,12 @@ export async function group(store?: Store | string, path: string | null = null,
  * @param chunkStore Store or path to directory in file system or name of zip file.
  * @param cacheAttrs If `true` (default), user attributes will be cached for attribute read operations
  *   If False, user attributes are reloaded from the store prior to all attribute read operations.
- *
+ * @param ipfsClient IPFS client
+ * @param cid IPFS CID
```

Comment:
ditto for this one IPFS Content Identifier (CID) which us a unique value used to identify files stored on the IPFS network. something like that